apps/apps.c

```diff
@@ -2082,11 +2082,19 @@ static int WIN32_rename(const char *from, const char *to)
                 tto[i] = (TCHAR)to[i];
     }
 
+#if defined(OPENSSL_SYS_WIN_CORE)
```

Comment:
I would rather see the goto duplicated and moved inside the if, so that pre-processor blocks don't split up code blocks.  for this hear, and the at line 2093 as well.

Reply:
Well, there actually is no reason to have this #ifdef. Originally it was kind of "fallback" for Win9x, but we don't need to care about that anymore. Since onecore.lib exports MoveFileExW alone, then that's what one should use. I mean one should just switch to MoveFileExW. It's available in since long enough not to care about its availability. So tfrom tto should be declared WCHAR and `if (sizeof(TCHAR) == 1)` - scraped.